docs/index.md

```diff
@@ -0,0 +1,48 @@
+# SpatialOS Unity GDK documentation
```

Comment:
if we rename it to index.md, it won't be shown by github as the main page of the docs folder

Reply:
I think we can write a pre-processing tool to convert Github format docs to Improbadoc format docs

Reply:
That sounds like it could be a good solution if you want to keep the GitHub docs browseable, though I guess it's another thing to maintain.

If you wanted to try putting something together, I've written up the steps you'd need to run through to get something running (and therefore the steps a tool would need to do): https://docs.google.com/document/d/1nCWButxwioEk4y7SlB76u4jDYkID4sXvAQzoDfmLCfM/edit#